clippy_lints/src/methods/mod.rs

```diff
@@ -2974,8 +2975,8 @@ declare_clippy_lint! {
     /// # let mut state = DefaultHasher::new();
     /// # let my_enum = Foo::Empty;
     /// match my_enum {
-    /// 	Empty => ().hash(&mut state),
-    /// 	WithValue(x) => x.hash(&mut state),
+    ///         Empty => ().hash(&mut state),
+    ///         WithValue(x) => x.hash(&mut state),
```

Comment:
nit: Can you change this to 4 spaces instead of 8? Seems unintentional

Same with the other change a few dozen lines down